src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.hpp

```diff
@@ -31,16 +31,19 @@ struct BrgemmKernelConfig : public snippets::KernelExecutorBase::GenericConfig {
 #endif
 private:
     dnnl_data_type_t dt_in0 {dnnl_f32}, dt_in1 {dnnl_f32};
-    char palette[64] = {};
     bool is_with_amx {false};
     bool is_with_comp {false};
     float beta {0};
     dnnl::impl::cpu::x64::cpu_isa_t isa {dnnl::impl::cpu::x64::isa_undef};
     dnnl_dim_t M {0}, N {0}, K {0}, LDA {0}, LDB {0}, LDC {0};
 };
 
+struct BrgemmCompiledKernel {
+    std::unique_ptr<dnnl::impl::cpu::x64::brgemm_kernel_t> compiled_kernel = nullptr;
+    char palette[64] = {};
```

Comment:
Let's add a comment here:
```suggestion
    // Note: Palette is treated as a part of a kernel because it is initialized during the kernel compilation stage. Each kernel need to store the pallet it was compiled with.
```

Reply:
Done, thanks!